### DISCUSSION TITLE
Can we please get htmx on the fediverse?

### DISCUSSION THREAD

**Discussion by aral** [bigskysoftware/htmx#2312] [General]
Some of us who are not on Elon Musk’s X would love to be able to get updates on htmx also (e.g., I missed https://github.com/bigskysoftware/htmx/discussions/2198#discussioncomment-8236851)

Could we please set up an account on the fediverse for htmx?

Ideally, of course, htmx could have its own fediverse instance, just for itself, at (e.g.) social.htmx.org but, given that involves time and effort, an account somewhere else would be easy to set up and, given what X has become, would be a good backup to have in any case (e.g., https://toot.io/ is an instance run by Jan, who also hosts my own server at https://mastodon.ar.al – he’s a good egg.)

_PS. I can see that the htmx account has ~40K followers on X and you might be hesitant of losing them, etc. It doesn’t have to be either/or. But even if it was, you will likely find that you will get far more engagement on the fediverse as it doesn’t have an algorithmic timeline. As a case study, when I closed my account on Twitter, I had ~42K followers. Now, on my own Mastodon instance, I also have 42K followers. The difference is that when I post, it shows up on the timelines of ~42K people. Whereas on X, you have no idea how many people will see your posts, etc.)_

**Comment by marcgr1** [comment]
i dont think mastodon should be taken seriously until it supports freedom of speech, that should be the standard regardless of whether you like musk or trump.
![Screenshot 2024-02-27 at 14 16 38](https://github.com/bigskysoftware/htmx/assets/78028419/ae117289-3fff-48e0-8fb4-1064b48c5d93)
look at point 5, who decides that?

**Comment by andryyy** [comment]
Sigh.

**Comment by aral** [comment]
Closing due to lack of feedback.
